revolutionpi/io_util.go

```diff
@@ -1,57 +1,64 @@
-package revolution_pi
+//go:build linux
+
+// Package revolutionpi implements the Revolution Pi.
+package revolutionpi
 
 func ioctlAddress(v int) int {
 	kbIocMagic := int('K')
 	magic := (((0) << (((0 + 8) + 8) + 14)) | ((kbIocMagic) << (0 + 8)) | ((v) << 0) | ((0) << ((0 + 8) + 8)))
 	return magic
 }
 
-const PICONTROL_NOT_CONNECTED = 0x8000
+// is the hex output received when a board is not connected.
```

Comment:
It strikes me as odd that the comment starts with "is" like this. No need to copy the name in again; I'd start with either "This is the hex output" or else just "The hex output"